### PR TITLE
docs(readme): update extract CSS example

### PR DESCRIPTION
**Pull request by DenRedsky**
Update readme for webpack v4+.

**What kind of change does this PR introduce?**

Readme fix.

**Did you add tests for your changes?**
Nope.

**If relevant, did you update the README?**
 Exactly.

**Summary**

Clarify information for current webpack version.

**Does this PR introduce a breaking change?**
Nope.

**Other information**

### CODE REVIEW

**Comment by DenRedsky** [comment]
@evilebottnawi review fixed, CLA accepted.

File: README.md [hunk 1]
```diff
@@ -462,35 +462,28 @@ module.exports = {
 
 ### Extract
 
-For production builds it's recommended to extract the CSS from your bundle being able to use parallel loading of CSS/JS resources later on. This can be achieved by using the [extract-text-webpack-plugin](https://github.com/webpack-contrib/extract-text-webpack-plugin) for webpack version  to extract the CSS when running in production mode.
-
-> :warning: Since `webpack >= 4.0.0` the `extract-text-webpack-plugin` should not be used for css. Use [mini-css-extract-plugin](https://github.com/webpack-contrib/mini-css-extract-plugin) instead.
```
Comment (evilebottnawi) [inline README.md]: Return warning, many people still use `ETWP` for css, thanks for PR!

**Comment by DenRedsky** [comment]
@evilebottnawi may I set to warning information about ETWP? And let example about mini-css as is?

**Comment by evilebottnawi** [comment]
@DenRedsky yep, only warning about ETWP for webpack@4

**Comment by DenRedsky** [comment]
Fix it soon

**Comment by michael-ciniawsky** [comment]
@DenRedsky Please rebase your PR against latest `master` when you have the time :)

**Comment by evilebottnawi** [comment]
Done in master, thanks for the PR

**Pull request closed without merging**